src/commons/services/utils.js

```diff
@@ -108,6 +108,35 @@
             return clusterName;
         }
 
+        vm.getVolumeDetails = function(clusterId) {
+            volumeList = [];
+            clusterData = vm.clusterData.clusters;
+            len = clusterData.length;
+            for ( i = 0; i < len; i++ ) {
+                clusterObj = clusterData[i];
+                for( key in clusterObj ) {
```

Comment:
checking the condition based on index for object properties is not best idea i guess ..
because may be response object's properties order can be change ..
may be in zero index can have "stats" properties .. developer should not depend on Object's properties order .